### PR TITLE
Architectural design

### PR DESCRIPTION
**Pull request by KaranSampath**

### CODE REVIEW

File: tasks/Architectural-design.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-> Process derived fromL: Woolley (2009): https://www.researchgate.net/profile/Anita-Woolley/publication/220520861_Means_vs_Ends_Implications_of_Process_and_Outcome_Focus_for_Team_Adaptation_and_Performance/links/567aa32e08ae19758380fc44/Means-vs-Ends-Implications-of-Process-and-Outcome-Focus-for-Team-Adaptation-and-Performance.pdf
+Woolley, A. W., Chabris, C. F., Pentland, A., Hashmi, N., & Malone, T. W. (2010). Evidence for a Collective Intelligence Factor in the Performance of Human Groups. Science, 330(6004), 686 LP – 688. https://doi.org/10.1126/science.1193147
+
+Supplementary information: https://science.sciencemag.org/content/sci/suppl/2010/09/29/science.1193147.DC1/Woolley_SOM_Revision_1.pdf
+ 
+Process derived from: Woolley (2009): https://www.researchgate.net/profile/Anita-Woolley/publication/220520861_Means_vs_Ends_Implications_of_Process_and_Outcome_Focus_for_Team_Adaptation_and_Performance/links/567aa32e08ae19758380fc44/Means-vs-Ends-Implications-of-Process-and-Outcome-Focus-for-Team-Adaptation-and-Performance.pdf
 
 # Stimuli
 ## The visual components
 > [to-do]
```
Comment (KaranSampath) [inline tasks/Architectural-design.md:13]: We'll need to find the visual components for the task before merging it.

File: tasks/Architectural-design.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+2. The participants were given scoring information. The scoring was purposefully complex to encourage trade-offs. 
+3. The room was video taped with the knowledge and consent of all participants.
+4. After the video was shown, teams were given 10 minutes to work on a worksheet. The worksheet was used to decide tasks, rough-draft building structure, and a division of roles. 
+5. Teams turned the worksheet in to the experimenter.
+6. Teams were given 40 minutes to complete building task.
+7. At 20 minutes, the team was told 20 minutes had past.
+8. At 35 minutes, the teams were told 5 minutes were remaining.
 
 
 ## Instructions
```
Comment (KaranSampath) [inline tasks/Architectural-design.md:31]: Instead of doing this verbatim, I think we can perhaps make this more concise and summarize it.

**Comment by KaranSampath** [comment]
Just wanted to check if we can move forward on this @bvikram03 @markwhiting

File: tasks/Architectural-design.md (hunk 2 above)
Comment (markwhiting) [reply tasks/Architectural-design.md:31]: I'd lean toward verbatim instructions when we have them. 

Ultimately we want to reproduce the underlying task if possible.

**Review by markwhiting — COMMENTED** [review]
Are there other questions holing this up?

**Review by bvikram03 — DISMISSED** [review]

**Review by bvikram03 — APPROVED** [review]

**Pull request merged**